api/src/main/java/org/openmrs/module/dhisreport/api/utils/MonthlyPeriod.java

```diff
@@ -38,13 +38,25 @@
     protected Date endDate;
 
     @Override
-    public Date getStart()
+    public void setStartDate( Date startDate )
+    {
+        this.startDate = startDate;
+    }
+
+    @Override
+    public void setEndDate( Date endDate )
+    {
+        this.endDate = endDate;
+    }
+
+    @Override
+    public Date getStartDate()
     {
         return startDate;
     }
 
     @Override
-    public Date getEnd()
+    public Date getEndDate()
```

Comment:
@maurya  I have take care of those changes. You can find those changes in this PR itself.